pyof/v0x04/controller2switch/multipart_request.py

```diff
@@ -62,7 +62,7 @@ def __init__(self, xid=None, multipart_type=None, flags=None, body=b''):
         super().__init__(xid)
         self.multipart_type = multipart_type
         self.flags = flags
-        self.body = body
+        self.body = BinaryData() if body is None else body
```

Comment:
`BinaryData()` -> `b''`; it's bad having GenericTypes when they are not needed.
The cleaner way is to set `b''` directly in the `__init__` method - same behavior, better readability.